src/UserSpaceInstrumentation/Trampoline.h

```diff
@@ -110,6 +111,39 @@ struct RelocatedInstruction {
                                                                        uint64_t old_address,
                                                                        uint64_t new_address);
 
+// Strictly speaking the max tempoline size is a compile time constant, but we prefer to compute it
+// here since this captures every change to the code constructing the trampoline.
+[[nodiscard]] uint64_t GetMaxTrampolineSize();
+
+// Creates a trampoline for the function at `function_address`. The trampoline is build at
+// `trampoline_address`. The trampoline will call `payload_address` with `function_address` as a
```

Comment:
Hmm,
function_address is the function we instrument. payload_address is the payload. We add instrumentation for the purpose of calling the payload. So that is what we need to call from the trampoline.

Reply:
What I meant is that `AppendPayloadCode` in `Trampoline.cpp` suggests that with "payload" you also mean the code in the trampoline that calls the payload, but from your comment it doesn't seem that that's intended. Apart from that, "payload" is slightly generic, it would be nice to find a term that hints better at "function called from the trampoline", but alas I don't have ideas. So yeah, that was my point. Maybe at least a comment on `AppendPayloadCode` (in addition to this one) could help.

Reply:
Clarified wording (AppendPayloadCode -> AppendCallToPayloadCode) and added a comment as suggested.